src/main/java/Products.java

```diff
@@ -0,0 +1,9 @@
+public class Products {
+    String name;
+    double price;
```

Comment:
Эти два поля могут быть константными:
```suggestion
    final String name;
    final double price;
```
Это исключит их изменение извне